src/rust/engine/src/context.rs

```diff
@@ -108,36 +106,35 @@ impl Core {
     let futures_timer_thread = Arc::new(futures_timer::HelperThread::new().unwrap());
     let fs_pool2 = fs_pool.clone();
     let futures_timer_thread2 = futures_timer_thread.clone();
-    let store_and_command_runner_and_http_client = Resettable::new(move || {
-      let local_store_dir = local_store_dir.clone();
-      let store = safe_create_dir_all_ioerror(&local_store_dir)
-        .map_err(|e| format!("Error making directory {:?}: {:?}", local_store_dir, e))
-        .and_then(|()| {
-          if remote_store_servers.is_empty() {
-            Store::local_only(local_store_dir, fs_pool2.clone())
-          } else {
-            Store::with_remote(
-              local_store_dir,
-              fs_pool2.clone(),
-              &remote_store_servers,
-              remote_instance_name.clone(),
-              &root_ca_certs,
-              oauth_bearer_token.clone(),
-              remote_store_thread_count,
-              remote_store_chunk_bytes,
-              remote_store_chunk_upload_timeout,
-              // TODO: Take a parameter
-              fs::BackoffConfig::new(Duration::from_millis(10), 1.0, Duration::from_millis(10))
-                .unwrap(),
-              remote_store_rpc_retries,
-              futures_timer_thread2.handle(),
+    let local_store_dir = local_store_dir.clone();
```

Comment:
This clone is no longer necessary, because we're now moving the value into a `FnOnce` closure not an `Fn` closure.